deployments/Dockerfile

```diff
@@ -1,21 +1,15 @@
 # This Dockerfile is used to build the image available on DockerHub
-FROM centos:centos7 as build
+FROM golang:1.15.13 as build
 
 # Add everything
 ADD . /usr/src/multus-cni
 
-ENV INSTALL_PKGS "git golang-1.15.13-0.el7.x86_64"
-RUN rpm --import https://mirror.go-repo.io/centos/RPM-GPG-KEY-GO-REPO && \
-    curl -s https://mirror.go-repo.io/centos/go-repo.repo | tee /etc/yum.repos.d/go-repo.repo && \
-    yum install -y $INSTALL_PKGS && \
-    rpm -V $INSTALL_PKGS && \
-    cd /usr/src/multus-cni && \
+RUN cd /usr/src/multus-cni && \
     ./hack/build-go.sh
 
-FROM centos:centos7
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 LABEL org.opencontainers.image.source https://github.com/k8snetworkplumbingwg/multus-cni
 COPY --from=build /usr/src/multus-cni /usr/src/multus-cni
 WORKDIR /
 
-ADD ./images/entrypoint.sh /
-ENTRYPOINT ["/entrypoint.sh"]
+ENTRYPOINT ["sleep"]
```

Comment:
Yep, I think this is OK for a default. We'll probably have a running process to do some of the "loopy" stuff that we do in the entrypoint, as we progress, that is...

Like, configuration regeneration: https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/images/entrypoint.sh#L426-L455

Also, something that we can actually omit -- and cover when we have the thick plugin architecture is this hacky PR from me that's not merged: https://github.com/k8snetworkplumbingwg/multus-cni/pull/686

It's a hacky way to update the secret, but in thick-plugin mode, we can just rely on client-go to update it for us, and we won't actually have to generate the secret anymore...

...for now though, let's keep this, just providing a bit of context too, as I look at it.

Reply:
> Yep, I think this is OK for a default. We'll probably have a running process to do some of the "loopy" stuff that we do in the entrypoint, as we progress, that is...
> 
> Like, configuration regeneration: https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/images/entrypoint.sh#L426-L455

Added it to the PR's description [TODO list](https://github.com/k8snetworkplumbingwg/multus-cni/pull/718#issue-724640335).

> 
> Also, something that we can actually omit -- and cover when we have the thick plugin architecture is this hacky PR from me that's not merged: #686
> 
> It's a hacky way to update the secret, but in thick-plugin mode, we can just rely on client-go to update it for us, and we won't actually have to generate the secret anymore...
> 
> ...for now though, let's keep this, just providing a bit of context too, as I look at it.

Resolving this conversation since I've tracked it in the PR's description TODO list.